### DISCUSSION TITLE
The sidebar directory of the documentation has disappeared.

### DISCUSSION THREAD

**Discussion by rikisamurai** [pmndrs/jotai#2890] [General]
The sidebar of the official Jotai website used to have a directory, but now it doesn't. It was very convenient to view the documentation in the previous way.
https://jotai.org/docs/core/atom
<img width="1245" alt="image" src="https://github.com/user-attachments/assets/8ec4a29e-f0f8-40da-aac8-cfedfc0bb4cd" />

**Comment by rikisamurai** [comment]
Could I expect this version of the documentation to come back?👀
<img width="722" alt="image" src="https://github.com/user-attachments/assets/118142a2-1f76-45f0-ad76-6d7341a6371f" />

**Reply by DavidJFelix** [reply]
If you click documentation there's a grid view:

![image](https://github.com/user-attachments/assets/ca2e7e7c-f54f-42fc-bf22-305d76d02923)

**Reply by dmaskasky** [reply]
Could we please bring the old layout back? Or at least make it an option?

**Reply by dai-shi** [reply]
I think @sandren has a plan for something.

**Reply by rikisamurai** [reply]
> If you click documentation there's a grid view:
> 
> ![image](https://private-user-images.githubusercontent.com/263432/399341801-ca2e7e7c-f54f-42fc-bf22-305d76d02923.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzU2MjQwNTAsIm5iZiI6MTczNTYyMzc1MCwicGF0aCI6Ii8yNjM0MzIvMzk5MzQxODAxLWNhMmU3ZTdjLWY1NGYtNDJmYy1iZjIyLTMwNWQ3NmQwMjkyMy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMjMxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTIzMVQwNTQyMzBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0zYTA0ZjAwN2Q4ZWQzMmVmMzIxOWJjOGFiNDU3Y2RkNmMxODg3NWUyOWFiNTA5YmVkYzY3Y2EwYzNiY2VkOGQzJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.ih8zSFhFUCWW2mZbCoUmcwhF284NrdGwayT72ppyL5g)

If I want to switch between different chapters of the documentation, it will be quite inconvenient. Previously, it only required one click (e.g., Async ➡️ SSR), but now it requires multiple clicks (e.g., Async ➡️ Grid View ➡️ SSR). Moreover, without the sidebar directory, it can be a bit confusing.
